clients/cobol-lsp-vscode-extension/src/services/CopybookFix.ts

```diff
@@ -15,7 +15,7 @@
 import * as vscode from "vscode";
 import { DownloadQueue } from "./DownloadQueue";
 
-export class CopybookResolver {
+export class CopybookFix {
```

Comment:
I'm not sure that this name is more descriptive, but it meets the naming convention at least 🤔

Reply:
well..the naming convention should be QuickFix right?

Reply:
I meant that "fix" is ambiguous here. Only after your explanation, I understood the real purpose of the class.

Reply:
Well, it's not QuickFix only there

Reply:
I think I need an on-demand explanation like Andrei.. so please use a better name or a comment  that clarify the intent